packages/zudoku/src/app/main.tsx

```diff
@@ -129,9 +129,13 @@ export const getRoutesByConfig = (config: ZudokuConfig): RouteObject[] => {
     {
       element: (
         <Zudoku {...options}>
-          {import.meta.env.IS_ZUPLO && import.meta.env.ZUPLO_BUILD_ID && (
-            <BuildCheck buildId={import.meta.env.ZUPLO_BUILD_ID} />
-          )}
+          <BuildCheck
+            buildId={
+              import.meta.env.IS_ZUPLO && import.meta.env.ZUPLO_BUILD_ID
+                ? import.meta.env.ZUPLO_BUILD_ID
+                : undefined
+            }
+          />
```

Comment:
I've made this change because for non-Zuplo projects we get this warning from Rollup/Vite:

> "BuildCheck" is imported from external module "zudoku/components" but never used in "src/A.tsx", "src/B.tsx", "src/C.tsx", ..., "../node_modules/zudoku/src/app/main.tsx" and "../node_modules/zudoku/src/app/entry.server.tsx".